commit 81fb956b221e4b1f919788d8a5f6a3a5462b86ae
Author: Øyvind Kolås <pippin@gimp.org>
Date:   Thu Mar 23 14:59:28 2017 +0100

    build: move G_LOG_DOMAIN define to a Makefile.am as suggested in glib docs...

diff --git a/configure.ac b/configure.ac
index 41ed17e1..36e8e2a1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1269,8 +1269,6 @@ done
 # We should support this at some point if possible
 #LDFLAGS="-Wl,-z,defs"
 
-CFLAGS="$CFLAGS -DG_LOG_DOMAIN=\\\"GEGL\\\""
-
 dnl bin/node-editors/Makefile
 AC_CONFIG_FILES([
 Makefile
diff --git a/gegl/Makefile.am b/gegl/Makefile.am
index 2030ebbe..b5f70d88 100644
--- a/gegl/Makefile.am
+++ b/gegl/Makefile.am
@@ -12,6 +12,7 @@ endif
 
 SUBDIRS = buffer graph module operation process property-types opencl
 
+
 CLEANFILES =
 
 AM_CPPFLAGS = \
@@ -31,6 +32,7 @@ AM_CPPFLAGS = \
 	-I$(top_builddir)/gegl/property-types \
 	-I$(top_srcdir)/gegl/property-types \
 	-DLIBDIR=\""$(libdir)"\" \
+        -DG_LOG_DOMAIN=\"GEGL\" \
 	-DGEGL_LOCALEDIR=\""$(GEGL_LOCALEDIR)"\"
 
 AM_CFLAGS = $(DEP_CFLAGS) $(BABL_CFLAGS)
